Examples/TcpReassembly/main.cpp

```diff
@@ -374,7 +373,7 @@ static void tcpReassemblyMsgReadyCallback(int8_t sideIndex, const pcpp::TcpStrea
 		if (result == 1)
 		{
 			// find the connection from the flow key
-			TcpReassemblyConnMgrIter iter2 = connMgr->find(flowKeyToCloseFiles);
+			auto iter2 = connMgr->find(flowKeyToCloseFiles);
```

Comment:
should we rename the variable to `flow2`, the same way we renamed `iter` to `flow`?

Reply:
Fixed in https://github.com/seladb/PcapPlusPlus/pull/1229/commits/48be196b84314896d8c55de42ac68a39470e5428